ui/js/controllers/tcjobactions.js

```diff
@@ -1,4 +1,5 @@
 "use strict";
+var jsonSchemaDefaults = require('json-schema-defaults');
```

Comment:
The syntax is fine, I would just define it a bit lower, inside the function (like around line 10).

If we wind up using json schema defaults all over the place (kind of doubt we would but I guess you never know) we could use a similar strategy as here and define an angular factory with the import:

https://github.com/mozilla/treeherder/blob/eeaa505a2f18fc858cdbcef2362c498786119d37/ui/js/services/main.js#L214